### ISSUE TITLE
Citizens freezing in place (ATM 9)

### ISSUE THREAD

**Issue by AZebra04** [ldtteam/minecolonies#10542]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Are you using the latest MineColonies Version?

- [X] I am running the latest **beta/release** version of MineColonies for my Minecraft version.
I am also running the latest versions of other mods that are part of my problem.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### What were you playing at the time? Were you able to reproduce it in both settings?

- [ ] Single Player
- [X] Multi Player

### Minecraft Version

1.20

### MineColonies Version

1.20.1 

### Structurize Version

1.20.1

### Related Mods and their Versions

There shouldn't be any mods affecting this 

### Current Behavior

All 21 of my colonists keep freezing in place after reloading my server. They do not do any work and are stuck even if recalled to their job or the town hall. I cannot find any errors in the logs stating why they stop moving. 

### Expected Behavior

The citizens should be moving as normal and completing tasks, but they are not. 

### Reproduction Steps

This issue just started happening over and over, sometimes to the point where restarting the server never unfreezes them at all. 

### Logs

[14:37:45] [Server thread/INFO] [minecolonies/]: Trying to spawn raid on colony with no loaded buildings, aborting! These logs still work, but nothing is happening within the colony.

### Anything else?

I have checked some subreddits but no luck on how to fix this issue

### Footer

--- AndyZebra
## Viewers
* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
Are you on the newest 1.20.1 minecolonies?

**Comment by AZebra04** [comment]
> Are you on the newest 1.20.1 minecolonies?

Yes I am, this is the latest release version for ATM 9 at the moment I believe

**Comment by Talyda** [comment]
ATM 9 is not Minecolonies - you need the latest minecolonies version. As far as I am aware, the version in ATM9 is problematic. Just check minecolonies version

**Comment by AZebra04** [comment]
> ATM 9 is not Minecolonies - you need the latest minecolonies version. As far as I am aware, the version in ATM9 is problematic. Just check minecolonies version

is there a way to fix this version within the ATM 9 pack? there is a lot of work put into this

**Comment by Raycoms** [comment]
Yes, just turn on content management and update minecolonies manually

**Comment by AZebra04** [comment]
> Yes, just turn on content management and update minecolonies manually

And what about the server? Can I replace the mod or is there another way to update it?

**Comment by Raycoms** [comment]
Yes, you'd have to do the same thing on the server

-------- Original Message --------
